lib/cpp/test/SecurityTest.cpp

```diff
@@ -75,13 +75,17 @@ struct GlobalFixture
             if (!boost::filesystem::exists(certFile("server.crt")))
             {
                 keyDir = boost::filesystem::path(master_test_suite().argv[master_test_suite().argc - 1]);
-                BOOST_REQUIRE_MESSAGE(boost::filesystem::exists(certFile("server.crt")),
-                                      "The last argument to this test must be the directory containing the test certificate(s).");
+                if (!boost::filesystem::exists(certFile("server.crt")))
+                {
+                	std::cerr << "The last argument to this test must be the directory containing the test certificate(s)." << std::endl;
```

Comment:
I agree with @hcorg, there must be a better alternative from boost test to do this.
the other questions is backward compatibility, miminum version is 1.53.0 right now.